docs/manual/guides/maintenance-template.de.md

```diff
@@ -15,7 +15,14 @@ Der Wartungsmodus lässt sich über den Menüpunkt Systemwartung aktivieren. Sob
 
 ![Contao Wartungsmodus](/de/guides/images/de/maintenance/wartungsmodus.jpg?classes=shadow)
 
-Im Core von Contao 4.9 sind für die Frontendausgabe im Wartungsmodus folgende Dateien zuständig:
+{{< version "4.13" >}}
+Der Wartungmodus beherrscht seit dieser Version drei Varianten:
+- Aktiviert man den Wartungsmodus im Contao-Manager, dann startet der Applikationswartungsmodus, bei dem Front- und Backend gesperrt sind. Es werden die generischen Wartungstemplates ausgespielt.
+- Aktivierung in der Seitenstruktur (je Seitenbaum getrennt) startet einen einfachen Wartungsmodus, bei dem nur das Frontend gesperrt ist. Auch hier werden die generischen Wartungstemplates ausgespielt.
+- Wenn bei Aktivierung in der Seitenstruktur auch jeweils ein Seite vom Typ 503 vorhanden ist, dann wird diese im Frontend angezeigt. Diese Seiten können redaktionell bearbeitet werden.
```

Comment:
There are only 2 modes:

* Global maintenance mode (via Command/Contao Manager)
* Per website root

And there are only 2 different ways of changing the output of the maintenance mode:

* Via `service_unavailable.html.twig` template (applies to both modes).
* Via 503 page type when using the second mode.